src/character.h

```diff
@@ -1998,7 +1998,7 @@ class Character : public Creature, public visitable
             return std::max( 1, focus_pool / 1000 );
         }
         void mod_focus( int amount ) {
-            focus_pool += amount * 1000;
+            focus_pool + amount * 1000 > 0 ? focus_pool += amount * 1000 : 1;
```

Comment:
This doesn't look like it does what you want, I think you want
```suggestion
            focus_pool = focus_pool + amount * 1000 > 0 ? focus_pool + amount * 1000 : 1;
```
Regardless, I think these are clearer and cleaner
```suggestion
            focus_pool += std::max( amount * 1000, 1 );
```
or
```suggestion
            focus_pool += amount * 1000;
            focus_pool = std::max( focus_pool, 0 );
```

Reply:
it does work due to focus_pol += amount*1000 being the result of the if but your 3rd version is cleaner. (the 2nd doesn't work due to mod_focus also handling negative amounts)

Reply:
Yeah, I commented a bit early and spent the next few minutes editing my reply. The second one didn't work, the third one is cleaner, as it is (well, was) if modifying focus would make it negative, nothing would happen.